readthedocs/rtd_tests/tests/test_views.py

```diff
@@ -213,71 +203,6 @@ def test_build_list_includes_external_versions(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn(external_version_build, response.context["build_qs"])
 
-    @mock.patch("readthedocs.projects.tasks.builds.update_docs_task")
-    @mock.patch("readthedocs.core.utils.cancel_build")
-    def test_rebuild_specific_commit(self, mock_update_docs_task, mock_cancel_build):
```

Comment:
All these deleted tests don't look to be specific to the old theme?

Reply:
These tests are for the POST request done to the dashboard to rebuild. That view was deleted and is now done via APIv3. That's why I deleted these tests as well.